edx_proctoring/templates/proctored_exam/ready_to_start.html

```diff
@@ -3,23 +3,29 @@
   <div>
     <h3>
     {% blocktrans %}
-      Follow these instructions
+      Important
     {% endblocktrans %}
     </h3>
     <p>
       {% blocktrans %}
-        &#8226; When you start your exam you will have {{ total_time }} to complete it. </br>
-        &#8226; You cannot stop the timer once you start. </br>
-        &#8226; If time expires before you finish your exam, your completed answers will be
-                submitted for review. </br>
-        &#8226; If you have not taken a proctored exam before using {{ platform_name }} please read
-                the exam rules and guidelines first. Reading these will ensure you don't accidentally violate
-                any of the proctored exam rules. {% endblocktrans %}
-                <a href="{{link_urls.online_proctoring_rules}}" target="_blank">
-                  {% blocktrans %}
-                  Read the {{ platform_name }} Online Proctoring Rules for Learners
-                  {% endblocktrans %}
-                </a> </br>
+        &#8226; You have {{ total_time }} to complete this exam. </br>
+        &#8226; You cannot stop the timer after you start the exam. </br>
+        &#8226; If time expires before you finish the exam, your completed answers will be submitted. </br>
```

Comment:
No problem. Thanks for pointing out the typo—it probably would have taken me a while to find that. :-)